native/packages/react-native-bpk-component-phone-input/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "react-native-bpk-component-phone-input",
+  "version": "0.0.0",
+  "main": "index.js",
+  "description": "Backpack React Native phone input component.",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:Skyscanner/backpack.git"
+  },
+  "author": "Backpack Design System <backpack@skyscanner.net>",
+  "license": "Apache-2.0",
+  "peerDependencies": {
+    "react": "^16.0.0-alpha.1",
```

Comment:
both react and native should be the same as the main package.json https://github.com/Skyscanner/backpack/blob/master/native/package.json#L22

Reply:
I believe we need to add a card to update these across the board

Reply:
Ok cool, I'll leave this for then.

Reply:
Do we have a card or do we need to create one?